SodaInventory/ClientApp/src/pages/Dashboard.js

```diff
@@ -48,23 +51,24 @@ class Dashboard extends Component {
         let cells = [];
         data.forEach((element) => {
             cells.push(
-                <TableRow key={element.id} onClick={() => this.setState({showItemModal: true})}>
-                    <TableCell>{element.item}</TableCell>
-                    <TableCell>{element.qty}</TableCell>
+                <TableRow key={element.itemId} onClick={() => this.setState({showItemModal: true})}>
+                    <TableCell>{element.itemName}</TableCell>
+                    <TableCell>{element.amount}</TableCell>
+                    <TableCell>{element.uom}</TableCell>
```

Comment:
I thought we discussed putting the unit of measure in the amount column...

Reply:
That's right! I totally forgot about this. I just made another commit to the PR that fixes this.